applications/CoSimulationApplication/CoSimulationApplication.json

```diff
@@ -1,7 +1,7 @@
 {
 	"wheel_name": "KratosCoSimulationApplication",
 	"included_modules": ["CoSimulationApplication"],
-	"included_binaries": ["KratosCoSimulationApplication.*", "KratosCoSimulationCore.*", "libKratosCoSimulationCore.*"],
+	"included_binaries": ["KratosCoSimulationApplication.*", "KratosCoSimulationCore.*", "libKratosCoSimulationCore.*", "co_sim_io.*"],
```

Comment:
is the MPI-binary included somewhere else?

Reply:
I am not sure to follow, what does this change has to do with mpi?

Reply:
Oh, I see you have a mpi extension. I can add it if you want but I would say that so far MPI support is still on transition. For example there is no trilinos and you can only work with MPICH/IntelMPI